index.js

```diff
@@ -0,0 +1,65 @@
+import Game from "./game.js"
+import HumanPlayer from "./humanPlayer.js"
+import ComputerPlayer from "./computerPlayer.js"
+import View from "./view.js"
+
+document.addEventListener("DOMContentLoaded", () => {
+  let robot = new ComputerPlayer("RoBot")
+  let human = new HumanPlayer(name)
+  let game = new Game({ referee: robot, guessee: human })
+
+  let el = document.querySelector("#hm")
+  new View(game, el)
+  const button = document.querySelector('input');
```

Comment:
Button and input form are different. If you do create a button tag in your html file, you'd need to give it an id and then grab it by that id in the index.js file.